src/Appwrite/CLI/Tasks.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Appwrite\CLI;
+
+use Utopia\Platform\Platform;
+
+class Tasks extends Platform
+{
+    public function __construct()
+    {
+        $this->addService('cliTasks', new TasksService());
+    }
+}
```

Comment:
Feels to me like the platform should be named `Appwrite` and `Appwrite` should have a `tasks` service, not `cliTasks`. WDYT?

Reply:
This will be a placeholder for other services.

Reply:
renamed platform to Appwrite and service to Tasks and moved both to new namespace `Appwrite\Platform`